string name #unique type identifier
string shape #circle or square (currently)
std_msgs/Float32 radius
std_msgs/Float32 width
#std_msgs/Float32 l
#std_msgs/Float32 w
